### PR TITLE
Add an ASCIIlotl to the boot screen

### PR DESCRIPTION
**Pull request by rmoff**
image

### CODE REVIEW

File: cmd/lakefs/cmd/run.go [hunk 1]
```diff
@@ -461,7 +461,7 @@ const runBanner = `
      ╚══════╝╚═╝  ╚═╝╚═╝  ╚═╝╚══════╝╚═╝     ╚══════╝
 
 │
-│ If you're running lakeFS locally for the first time,
+│ If you're running lakeFS locally for the first time,
```
Comment (nopcoder) [inline cmd/lakefs/cmd/run.go:464]: can you trim the leading spaces/tab at the end of this line

File: cmd/lakefs/cmd/run.go [hunk 2]
```diff
@@ -471,8 +471,8 @@ const runBanner = `
 │
 
 │
-│ For support or any other question,
-│     join our Slack channel https://docs.lakefs.io/slack
+│ For support or any other question,                            >(.＿.)<
```
Comment (nopcoder) [inline cmd/lakefs/cmd/run.go:474]: same here - from the diff it looks like there is a tab at the end

**Review by nopcoder — CHANGES_REQUESTED** [review]

**Review by nopcoder — APPROVED** [review]
Great 😃

**Comment by ozkatz** [comment]
I love everything about this PR.

**Comment by arielshaqed** [comment]
Great PR but mislabelled.  This needs to be `include-changelog` (and possibly headline the release).

**Pull request merged**